src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,41 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonRowInfo } from './PersonRowInfo';
+
+interface Props {
+  people: Person[];
+  // selectedSlug: string,
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  // const { people, selectedSlug } = props;
```

Comment:
Comment is not needed here as it's clear enough what the code is doing. Please remove it. 🧹